compiler/src/iree/compiler/ExternalInterfaces/LinalgExtExternalModels.cpp

```diff
@@ -123,14 +123,15 @@ void registerOpsWithLinalgExtOpInterface(mlir::MLIRContext *context) {
 } // namespace
 
 void registerLinalgExtExternalModels(DialectRegistry &registry) {
-  registry.addExtension(+[](MLIRContext *ctx,
-                            IREE::LinalgExt::IREELinalgExtDialect *dialect) {
-    ctx->loadDialect<mlir::linalg::LinalgDialect>();
-
+  registry.addExtension(+[](MLIRContext *ctx, linalg::LinalgDialect *dialect) {
 #define GET_OP_LIST
     registerOpsWithLinalgExtOpInterface<
 #include "mlir/Dialect/Linalg/IR/LinalgStructuredOps.cpp.inc"
         >(ctx);
+  });
+  registry.addExtension(+[](MLIRContext *ctx,
+                            IREE::LinalgExt::IREELinalgExtDialect *dialect) {
+    ctx->loadDialect<linalg::LinalgDialect>();
     linalg::SoftmaxOp::attachInterface<SoftmaxFusionOpInterfaceAdapter>(*ctx);
```

Comment:
Why isn't softmax registered with the rest of linalg?

Reply:
Oh, my bad I forgot Softmax was part of linalg, not LinalgExt

Reply:
Ah ok I was worried there was some insane MLIR wizardry that made different ops need different registration.